doc/classes/String.xml

```diff
@@ -313,48 +337,49 @@
 			<argument index="1" name="placeholder" type="String" default="{_}">
 			</argument>
 			<description>
+				Formats the string by replacing all occurences of [code]placeholder[/code] with [code]values[/code].
 			</description>
 		</method>
 		<method name="get_base_dir">
 			<return type="String">
 			</return>
 			<description>
-				If the string is a path to a file, return the base directory.
+				If the string is a path to a file, returns the base directory.
 			</description>
 		</method>
 		<method name="get_basename">
 			<return type="String">
 			</return>
 			<description>
-				If the string is a path to a file, return the path to the file without the extension.
+				If the string is a path to a file, returns the path to the file without the extension.
```

Comment:
If the string is a valid file path, returns the full file path without the extension.